src/fixate/__init__.py

```diff
@@ -30,6 +30,6 @@
     generate_relay_matrix_pin_list as generate_relay_matrix_pin_list,
 )
 
-from fixate.__main__ import run_main_program as run
+from fixate.main import run_main_program as run
```

Comment:
What does this import achieve?

Reply:
It exports the function to be part of the public api and allows creating script to be directly executed. Similar to what I've done in `atdev`, but at the fixate level. Once this is released, I'll use this (and the `--diagnositic-log-dir` argument) to fix the hackiness in `atdev`. Josh added an example also:
https://github.com/PyFixate/Fixate/blob/master/src/fixate/examples/tiny.py#L45C1-L49C31

I'd like to slowly make it clearer what is public api vs private implementation detail.